src/tui/rendering.rs

```diff
@@ -258,6 +261,9 @@ impl Tui {
     ) {
         let selected_process = search_results.nth(self.get_selected_row_index());
         let lines = process_details_lines(selected_process);
+
+        self.update_process_details_number_of_lines(area, selected_process);
```

Comment:
I'm unsure if updating the state variable during rendering smells, but I couldn't figure out how to do this correctly.

Reply:
I don't have any better idea so lets go with it for now.